packages/core/src/test/codewhisperer/commands/transformByQ.test.ts

```diff
@@ -131,25 +131,18 @@ describe('transformByQ', function () {
         }, NoOpenProjectsError)
     })
 
-    it('WHEN stop job called with invalid jobId THEN throws error', async function () {
-        await assert.rejects(async () => {
-            await stopJob('')
-        }, Error)
+    it('WHEN stop job called with invalid jobId THEN stop API not called', async function () {
+        const stopJobStub = sinon.stub(codeWhisperer.codeWhispererClient, 'codeModernizerStopCodeTransformation')
+        await stopJob('')
+        sinon.assert.notCalled(stopJobStub)
     })
 
     it('WHEN stop job called with valid jobId THEN stop API called', async function () {
-        transformByQState.setToRunning()
         const stopJobStub = sinon.stub(codeWhisperer.codeWhispererClient, 'codeModernizerStopCodeTransformation')
         await stopJob('dummyId')
         sinon.assert.calledWithExactly(stopJobStub, { transformationJobId: 'dummyId' })
     })
 
-    it('WHEN stop job that has not been started THEN stop API not called', async function () {
```

Comment:
Do we not need this edge case / is it tested elsewhere?

Reply:
No longer needed since in `stopJob` I removed a check that checks if the job is marked as "running" or not, since that was a redundant check as it's handled elsewhere.